relay-server/src/actors/processor.rs

```diff
@@ -1507,8 +1681,22 @@ impl EnvelopeProcessorService {
         let upstream = self.inner.config.upstream_descriptor();
         let dsn = PartialDsn::outbound(&scoping, upstream);
 
-        for (partition_key, buckets) in partition_buckets(scoping.project_key, buckets, partitions)
-        {
+        let _bucket_qty = buckets.len();
+        let bucket_partitions = partition_buckets(scoping.project_key, buckets, partitions);
+
+        #[cfg(feature = "processing")]
```

Comment:
I think it makes sense to check the cached rate limits even on non-processing relays

Reply:
well atm, they just get updated from redis, which isnt available on non-processing relays, so im not sure if that would make sense

Reply:
I am not sure how propagation of quotas/limits to pops is exactly working, but I don't see a reason why we cant check it now and once we actually have a working propagation everything just works